development/image-builder/config.go

```diff
@@ -27,6 +28,9 @@ type Config struct {
 	// SignConfig contains custom configuration of signers
 	// as well as org/repo mapping of enabled signers in specific repository
 	SignConfig SignConfig `yaml:"sign-config" json:"sign-config"`
+	// PreBuildScript contains path to custom script that will be executed before build
+	// As first parameter it will receive docker tag generated from TagTemplate
+	PreBuildScript string `yaml:"pre-build-script" json:"pre-build-script"`
```

Comment:
I do not really understand why you've decided to add this part of code here. These kinds of stuff should not be defined in global configuration file, as they are specific only to the specific job. Config file should contain configuration only applicable on global level. I'd suggest defining this as program argument, which can be defined in the ProwJob. Take a look at the `--tag` value and how people can define their own tag templates.